mod/vecevent.mod

```diff
@@ -1,7 +1,66 @@
-:  Vector stream of events
+COMMENT
```

Comment:
Let's undo these changes. I copied it from PyNN, I'm not exactly sure if their license allows us to copy

Reply:
So rebase?